### PR TITLE
Add theme support and two optional themes

### PR DESCRIPTION
**Pull request by f**
I added `theme` support to the web UI and two optional themes.

- *By default, theme is set to `modern`*.
- The original old theme called `storm`.
- You can switch themes via query string (e.g. http://localhost:9002/?theme=storm will override the theme)
- `storm ssh --theme storm` will start storm web ui with the `storm` theme.

### New Black Theme
Black

### New Modern Theme (Default)
Modern

### CODE REVIEW

**Comment by f** [comment]
Any updates on this?

File: storm/web.py [hunk 1]
```diff
@@ -9,13 +9,18 @@
...
+def render(template, theme):
     static_dir = os.path.dirname(os.path.abspath(__file__))
     path = os.path.join(static_dir, 'templates', template)
     with open(path) as fobj:
         content = fobj.read()
+
+    if 'theme' in request.args:
+        theme = request.args.get('theme')
+
+    content = content.replace("__THEME__", theme)
```
Comment (berkerpeksag) [inline storm/web.py]:
```py
content = content.replace('__THEME__', request.args.get('theme', theme))
```

File: storm/__main__.py [hunk 2]
```diff
@@ -295,11 +295,12 @@ def backup(target_file, config=None):
 
 @command('web')
 @arg('port', nargs='?', default=9002, type=int)
+@arg('theme', nargs='?', default="modern")
```
Comment (berkerpeksag) [inline storm/__main__.py]: It would be nice to add `choices=['modern', 'black', 'storm']`.

**Comment by emre** [comment]
Thanks.

**Pull request merged**